src/data/abilities/ab-attrs/sturdy-ab-attr.ts

```diff
@@ -5,10 +5,16 @@ import type { Pokemon } from "#field/pokemon";
 import type { Move } from "#moves/move";
 import type { NumberHolder } from "#utils/common-utils";
 
-export class PreDefendFullHpEndureAbAttr extends PreDefendAbAttr {
+/**
+ * If the pokemon with this ab-attr is full hp and hit with a move that would 1-hit faint it, it will survive with 1 hp left.
+ * Wonder Guard _overrules_ this ab-attr.
```

Comment:
```suggestion
 * If the pokemon with this ab-attr is full hp and hit with a move that would 1-hit faint it, it will survive with 1 hp left (_unless it also has the ability Wonder Guard_).
```
It's not immediately clear what "overruling" the attribute entails. You may want to break this up to reduce line length as well.

Reply:
```suggestion
 * If the pokemon with this `AbAttr` is full HP and is hit with a move that would 1-hit faint it,
 * it will survive with 1 HP left (_unless it also has the ability Wonder Guard_).
```
Modified inner's suggestion to break it into 2 lines.